src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard-v3/dashboard/dashboard-v3.component.ts

```diff
@@ -98,8 +102,17 @@ export class DashboardV3Component extends PrometheusListHelper implements OnInit
     });
     this.getPrometheusData(this.lastHourDateObject);
     this.getDetailsCardData();
+    this.getTelemetryReport();
+    this.telemetryURL = 'https://telemetry-public.ceph.com/';
   }
 
+  getTelemetryText(): string {
+    return this.telemetryEnabled
+      ? 'Cluster telemetry is enabled'
+      : 'Cluster telemetry is disabled. To enbale the Telemetry, \
+       click settings icon on top navigation bar and select \
+       Telemetry configration.';
```

Comment:
we don't need the backslashes right? I remember it used to work without the backslashes for new lines.

```suggestion
      : 'Cluster telemetry is disabled. To enbale the Telemetry,
       click settings icon on top navigation bar and select
       Telemetry configration.';
```

Reply:
Doesn't work without it.